src/main/scanImages.test.ts

```diff
@@ -1,40 +1,23 @@
 import 'jest-extended';
-import md5 from 'md5';
 import * as LocalMain from '@getflywheel/local/main';
 import { createMockServiceContainer } from '../test/mockCreators';
 import { scanImagesFactory } from './scanImages';
 import { createStore } from './createStore';
 
+
 const sitePath = '/Users/cool-man-joe/Local Sites/twice-baked-potato';
 const serviceContainer = createMockServiceContainer(sitePath);
 
 const imageDataStore = createStore();
 
-/**
- * @todo Clean up dependencies that are unused and make fs-extra a dev only dependency
- */
-
-jest.mock('fs-extra');
-const fs = require('fs-extra');
-fs.statSync.mockImplementation((filePath: string) => ({
-	size: Math.floor(Math.random() * 99999) + 700
+jest.mock('./errorReporting', () => ({
+	reportScanRequest: jest.fn(),
```

Comment:
Did jest not automatically mock these functions out for you?

Reply:
Jest struggled to mock these out initially, but this extra specificity is no longer needed. Good catch!